components/log-viewer-webui/client/src/pages/IngestPage/sqlConfig.ts

```diff
@@ -0,0 +1,71 @@
+import axios from "axios";
+
+
+/**
+ * Sql table names.
+ */
+const SQL_CONFIG = Object.freeze({
+    SqlDbClpArchivesTableName: "clp_archives",
+    SqlDbClpFilesTableName: "clp_files",
+    SqlDbCompressionJobsTableName: "compression_jobs",
+});
+
+/**
+ * Enum of the column names for the `clp_archives` table.
+ *
+ * @enum {string}
+ */
+const CLP_ARCHIVES_TABLE_COLUMN_NAMES = Object.freeze({
+    BEGIN_TIMESTAMP: "begin_timestamp",
+    END_TIMESTAMP: "end_timestamp",
+    UNCOMPRESSED_SIZE: "uncompressed_size",
+    SIZE: "size",
+});
```

Comment:
let's replace it with a TS enum
```suggestion
/**
 * Column names for the `clp_archives` table.
 */
enum CLP_ARCHIVES_TABLE_COLUMN_NAMES {
    BEGIN_TIMESTAMP = "begin_timestamp",
    END_TIMESTAMP = "end_timestamp",
    UNCOMPRESSED_SIZE = "uncompressed_size",
    SIZE = "size",
}
```